README.md

```diff
@@ -93,6 +97,20 @@ The basic operation of the tool is this:
 The data can then be loaded by the ilib-localedata package for use in the various ilib-*
 packages.
 
+## Basic Operation for legacy version of ilib
+1. Write the `inc` file to specify which files to assemble.
+   If not, the default file that the tool (`./src/ilib-ilib-assemble-inc.js`)has will be used.
+2. Read the javscript files from the inc file list and extract the list of dependent JS and locale data.
+3. After the tool has been executed, the two style files will be placed the `output-dir`.
+   One is for the result of all JS files merged into a single usable file. If the file name is not specified, the default file name is `ilib-assemble.js`. and  another file is for the locale data per `[locale].js`.  
```

Comment:
If you change the default file name below, then remember to fix this documentation as well

Reply:
yep. I also updated it